config_models/models.py

```diff
@@ -131,7 +131,7 @@ def current(cls, *args):
         """
         cache_key = cls.cache_key_name(*args)
         cached_response = TieredCache.get_cached_response(cache_key)
-        if cached_response.is_found:
+        if cached_response.is_found and cached_response is not None:
```

Comment:
[inform] The None check was there before at this place. But it was removed when `TieredCache` was included.

https://github.com/openedx/django-config-models/pull/55/files#diff-89972c71903f4383220fc6ece46180b9c5c52f3467b07de6cc662ae9a546f49aL199-L201